.circleci/config.yml

```diff
@@ -41,6 +41,14 @@ jobs:
           name: Test Secret Ballot
           working_directory: ballot
           command: oasis-chain > oasis-chain.log & oasis test
+      - run:
+	  name: Build Rock Paper Scissors
```

Comment:
@nhynes like this?

Reply:
seems reasonable except for the tabs instead of spaces :)

Reply:
okie dokie